ardupilot/equipment/power/20010.BatteryCells.uavcan

```diff
@@ -0,0 +1,7 @@
+#
+# Battery cell voltages
+# Rate: set by parameter on smart battery (default off)
+#
+
+float16[<=24] voltages            # [Volt]
+uint8 index                       # Cell block index, index 0 corresponds to cells 0 - 23, index 1 cells 24 - 48, etc

```

Comment:
maybe uint16 and make the index be the number of the first cell

Reply:
that would be a small price to pay for more flexibility, also a little bit more intuitive to use